src/components/container/ConnectGarmin/ConnectGarmin.stories.tsx

```diff
@@ -38,4 +38,16 @@ export const NotEnabledHide = Template.bind({});
 NotEnabledHide.args = { previewState: "notEnabled", title: "Garmin", disabledBehavior: "hide" };
 
 export const NotEnabledDisplayError = Template.bind({});
-NotEnabledDisplayError.args = { previewState: "notEnabled", title: "Garmin", disabledBehavior: "displayError" };
\ No newline at end of file
+NotEnabledDisplayError.args = { previewState: "notEnabled", title: "Garmin", disabledBehavior: "displayError" };
+
+export const HideConnectedNotConnected = Template.bind({});
+HideConnectedNotConnected.args = { previewState: "notConnected", title: "Garmin", hideWhenConnected: true };
+
+export const HideConnectedUnauthorized = Template.bind({});
+HideConnectedUnauthorized.args = { previewState: "unauthorized", title: "Garmin", hideWhenConnected: true };
+
+export const HideConnectedFetchComplete = Template.bind({});
+HideConnectedFetchComplete.args = { previewState: "fetchComplete", title: "Garmin", hideWhenConnected: true };
```

Comment:
nit; the new stories here are nearly the same as fitbit, but the order is different.